src/main/resources/migration/common/V12__rename_assessment_to_offender.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE assessment RENAME TO offender;
+
+ALTER TABLE offender ADD COLUMN first_name VARCHAR(35);
+ALTER TABLE offender ADD COLUMN last_name VARCHAR(35);
+ALTER TABLE offender ADD COLUMN hdced DATE;
```

Comment:
Can we add not null?

Reply:
Yes we can because we only create records for offenders with an HDCED, updated